collector/system_collector.go

```diff
@@ -82,9 +84,20 @@ func (sc *systemCollector) Collect(ch chan<- prometheus.Metric) {
 		ch <- prometheus.MustNewConstMetric(sc.clusterNodeStatus, prometheus.GaugeValue, nm.Status, nm.IPAddress, nm.Type)
 	}
 
-	serviceMetrics := sc.collectClusterServiceMetrics()
+	var serviceMetrics []systemStatusMetric
+	serviceMetrics = append(serviceMetrics, sc.collectApplianceServiceMetric())
+	serviceMetrics = append(serviceMetrics, sc.collectMessageBusServiceMetric())
+	serviceMetrics = append(serviceMetrics, sc.collectNTPServiceMetric())
+	serviceMetrics = append(serviceMetrics, sc.collectUpgradeAgentServiceMetric())
+	serviceMetrics = append(serviceMetrics, sc.collectProtonServiceMetric())
+	serviceMetrics = append(serviceMetrics, sc.collectProxyServiceMetric())
+	serviceMetrics = append(serviceMetrics, sc.collectRabbitMQServiceMetric())
+	serviceMetrics = append(serviceMetrics, sc.collectSNMPServiceMetric())
+	serviceMetrics = append(serviceMetrics, sc.collectSSHServiceMetric())
+	serviceMetrics = append(serviceMetrics, sc.collectSearchServiceMetric())
+	serviceMetrics = append(serviceMetrics, sc.collectSyslogServiceMetric())
 	for _, svm := range serviceMetrics {
-		ch <- prometheus.MustNewConstMetric(sc.applianceManagementServiceStatus, prometheus.GaugeValue, svm.Status)
+		ch <- prometheus.MustNewConstMetric(sc.systemServiceStatus, prometheus.GaugeValue, svm.Status, svm.Name)
```

Comment:
what happens if `svm` is an empty object?
when the collect failed, it will return empty object